tests/assets/analytics.test.js

```diff
@@ -37,16 +37,21 @@ describe(__filename, () => {
         eventAction: action,
         eventLabel: label,
       });
-      expect(Array.from(window.dataLayer.push.mock.calls[0][0])).toEqual([
-        'event',
-        category,
-        {
+      // Ugly check but the content of the object is not the only thing that
+      // matters: pushing an array doesn't work.
+      expect(window.dataLayer.push.mock.calls[0][0].toString()).toBe(
+        '[object Arguments]'
+      );
+      expect(window.dataLayer.push.mock.calls[0][0]).toMatchObject({
```

Comment:
I don't know if we need to change this assertion if we know that we pass an array-like object above.

Reply:
You're right - I rewrote it multiple times hoping to find an elegant way of catching this, but the additional `.not.toBeInstanceOf(Array);` is better.